README.md

```diff
@@ -47,26 +47,19 @@ npm start
 
 #### Trading
 
-* `CONNECTOR_LEDGERS` (default: `[]`) Ledgers where this connector has accounts. Used to auto-generate `CONNECTOR_PAIRS`.
-```js
-[
-  "USD@example.usd-ledger.",
-  "EUR@example.eur-ledger."
-]
-```
-* `CONNECTOR_CREDENTIALS` (default: `{}`) Connector's login credentials for various ledgers, ex.
+* `CONNECTOR_LEDGERS` (default: `{}`) Connector's login credentials for ledgers where it has accounts. Used to auto-generate `CONNECTOR_PAIRS`.
 ```js
 {
   // Using Basic Auth
-  "<ledger_address>": {
+  "USD@example.usd-ledger.": {
```

Comment:
I definitely agree that the `asset@ledger` format is weird. Adding `currency` (or `_currency` to the object could mean that it gets passed into the plugin options even though the plugin shouldn't care what the connector calls its asset.